src/primevue/autocomplete/autocomplete.ts

```diff
@@ -0,0 +1,42 @@
+import { AutoCompletePassThroughOptions } from "primevue/autocomplete";
+import { tw } from "@/lib/tags.ts";
+import { base, small } from "../inputText/inputText";
```

Comment:
Suggestion: here I would actually prefer to have some copy paste rather than creating a dependency between two different components.